subprojects/docs/src/samples/testing/jacoco/quickstart/kotlin/build.gradle.kts

```diff
@@ -33,7 +33,7 @@ tasks.test {
 
 
 // tag::report-configuration[]
-tasks.jacocoTestReport {
+tasks.named<JacocoReport>("jacocoTestReport") {
```

Comment:
:x: `tasks.jacocoTestReport {` is already lazy

Reply:
Revert is pending.